src/edu/stanford/nlp/ling/tokensregex/parser/ParseException.java

```diff
@@ -89,7 +89,7 @@ private static String initialise(Token currentToken,
                            int[][] expectedTokenSequences,
                            String[] tokenImage) {
     String eol = System.getProperty("line.separator", "\n");
-    StringBuffer expected = new StringBuffer();
+    StringBuilder expected = new StringBuilder();
```

Comment:
You're updating a bunch of autogen code...  it will just get undone the next time these tools get updated.  I'm pretty averse to separating this out myself

Reply:
Oh, any hint about those tools, how can one run/change them?